tests/unit/test_bundle_updates.py

```diff
@@ -0,0 +1,36 @@
+import json
+import unittest
+
+from mock import patch
+
+from dcpquery import config
+from dcpquery.etl import drop_one_bundle, process_bundle_event
+from tests import mock_bundle_deletion_event
+
+
+class BundleUpdateEvewnts(unittest.TestCase):
+    def test_drop_one_bundle_handles_deletion(self):
+        bundle_uuid = 'a2f76df2-a80f-4c66-8fdd-169ae350be4e'
+        bundle_version = '2018-10-11T220441.424590Z'
+        bundle_fqid = bundle_uuid + '.' + bundle_version
+        file_count = config.db_session.execute("SELECT COUNT(*) from files;").fetchall()[0][0]
+        assert config.db_session.execute(
+            f"SELECT COUNT(*) FROM bundles WHERE fqid='{bundle_fqid}'").fetchall()[0][0] == 1
+        assert config.db_session.execute(
+            f"SELECT COUNT(*) FROM bundle_file_links where bundle_fqid='{bundle_fqid}'").fetchall()[0][0] == 50
+        drop_one_bundle(bundle_uuid, bundle_version)
+        assert config.db_session.execute(
+            f"SELECT COUNT(*) FROM bundles WHERE fqid='{bundle_fqid}'").fetchall()[0][0] == 0
+
+        assert config.db_session.execute(
+            f"SELECT COUNT(*) FROM bundle_file_links where bundle_fqid='{bundle_fqid}'").fetchall()[0][0] == 0
+
+        post_deletion_file_count = config.db_session.execute("SELECT COUNT(*) from files;").fetchall()[0][0]
+        assert post_deletion_file_count < file_count
```

Comment:
Please rewrite the above asserts using unittest helper assertions, which provide more information when the assertion fails.